projects/elife.yaml

```diff
@@ -769,6 +769,19 @@ elife-bot:
             # elife-bot-event-property--prod, elife-bot-event-property--end2end, etc.
             - elife-bot-event-property--{instance}
     aws-alt:
+        s1604:
```

Comment:
this is the same as `standalone` so correct solution. We should consider dropping it to just `s3: false`? It's not very useful to create buckets when the goal is to test Salt formulas

Reply:
it was honestly just a face-roll of a change to get something working. 

I agree though. If highstate in standalone doesn't require those buckets to be there, get rid of them